src/test/fuzz/fuzz.cpp

```diff
@@ -19,8 +19,6 @@ static bool read_stdin(std::vector<uint8_t>& data)
     ssize_t length = 0;
     while ((length = read(STDIN_FILENO, buffer, 1024)) > 0) {
         data.insert(data.end(), buffer, buffer + length);
-
-        if (data.size() > (1 << 20)) return false;
     }
     return length == 0;
```

Comment:
Why is `length == 0` invalid? Seems overly restrictive.

Reply:
It is the other way around: `length != 0` is invalid. `length == 0` means that end of file was reached which is what we want :)